src/mvt_getter.py

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
+import json
+from mapcss import MapCSS
+import os
+import sys
+import math
+from optparse import OptionParser
+
+reload(sys)
+sys.setdefaultencoding("utf-8")  # a hack to support UTF-8
+
+
+def pixel_size_at_zoom(z, l=1):
+    """
+    Converts l pixels on tiles into length on zoom z
+    """
+    return int(math.ceil(l * 20037508.342789244 / 512 * 2 / (2 ** z)))
+
+
+def get_vectors(minzoom, maxzoom, x, y, style, vec):
+    geomcolumn = "way"
+
+    pxtolerance = 1.8
```

Comment:
should be <= 1. because Mapbox overzooms tiles up to 1024, that should be around 0.5